frontend/public/locales/en/learn.json

```
@@ -1,6 +1,12 @@
 {
   "header": "Learn",
   "explore": "Explore our content",
+  "breadcrumbs": [
+    {
+      "text":"Retirement Hub",
+      "link":"/home"
+    }
+  ],
```

Return an object and explicitly use the key when you are generating the breadcrumb items.

```suggestion
  "breadcrumbs":  {
  "home": {
      "text":"Retirement Hub",
      "link":"/home"
    }
  },
```

```tsx
breadcrumItems: { [
{link: t("breadcrumbs.home.link"), text:, t("breadcrumbs.home.text")}
]}
```

---

Fixed this